### ISSUE TITLE
Write an HTML axe `violations` reporter

### ISSUE THREAD

**Issue by 1Copenut** [1Copenut/paus-tools#9]
## Description
I'd like to have a way to ingest a JSON or text file and have it generate an HTML report of a11y violations. This shouldn't be too hard with a simple web app.

## MVP
* [ ] Allows user to upload a file (potentially dangerous, but would make the tool more useful)
* [ ] Parse the file and render HTML. Consider your [ui-from-json](https://github.com/1Copenut/vanilla-accessibility-js/tree/main/ui-from-json-fetch) vanilla JS project as a starting point.
* [ ] Make it accessible!
